teaser/logic/buildingobjects/calculation/four_element.py

```diff
@@ -288,6 +288,8 @@ class FourElement(object):
         the ambient.
     ua_value_win : float [W/K]
         U-Value times outer wall area.
+    u_value_win : float [W/(m2K)]
+        U-Value of windows.
```

Comment:
My understanding is, that this is the area-weighted U value of all windows. If that is the meaning of this value we might want to explain that.